trail/query/views.py

```diff
@@ -61,3 +66,162 @@ def print_results(request):
 
     return render(request, "query.html",
                   {"data": query_results, "wcsdata": wcs_list, "queryform": form, "render_table": True})
+
+
+class MetadataDAO(APIView):
+    """
+     Data Accessing objects for metadata, all query for metadatas are implemented here
+
+    Notes
+    -----
+    """
+
+    RALOW = "raLow"
+    RAHIGH = "raHigh"
+    DECLOW = "decLow"
+    DECHIGH = "decHigh"
+
+    """Support Query that returns metadata entry as a result through metadata info
+
+    Notes
+    -----
+    """
+    # gets all the metadata entry that matches the query parameters.
+    def getMetadataByParams(self, paramDict):
+
+        """
+        Returns the Metadata entries that meets the users input
+        Parameters
```

Comment:
- Please add a blank line before Parameters
- You list `fields` and `queryParam` as Parameters, but I only see `paramDict`?
- Missing a Returns docstring section